package.json

```diff
@@ -12,9 +12,10 @@
   "dependencies": {
     "axios": "^0.21.1",
     "dotenv": "^8.2.0",
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "kill-port": "^2.0.1"
```

Comment:
we don't need to update package json for this. we can just update devcontainer file with flag to auto-install this dependency.

Reply:
Issue is been addressed by adding "CI=true npx kill-port --yes " in devcontainer.json file